service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/respondentsolicitor/UnregisteredRespondentSolicitorTemplate.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.notify.respondentsolicitor;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+
+@Value
+@Builder
+public class UnregisteredRespondentSolicitorTemplate implements NotifyData {
+    String ccdCode;
```

Comment:
very minor: maybe ccdCaseNumber or something else (dont like ccdCode, but not sure why)

Reply:
this is what was in the template, I can change the template if you want it to match `ccdCaseNumber` or just add a json property annotation and change the field name.

Reply:
Changed to ccdNumber, consistent with the NoC emails